### PR TITLE
[favicons] replace files, add svg one for dark theme 

### PR DESCRIPTION
**Pull request by vince19972**
Related: https://github.com/allinbits/design/issues/293

For replacing favicons and adding `svg` one for dark theme

- [ ] ~~Image https://deploy-preview-70--cosmos-network.netlify.app/img/icons/apple-touch-icon-152x152.png is ill-formed~~
- [x] Image https://deploy-preview-70--cosmos-network.netlify.app/apple-touch-icon-precomposed.png is ill-formed
- [ ] ~~The mask icon for Safari pinned tabs is ill-formed~~

Preview: https://deploy-preview-70--cosmos-network.netlify.app

### CODE REVIEW

**Comment by lovincyrus** [comment]
Deploy Preview was blocked by an unrecognized author :D Just accepted the request on Netlify! We should be able to test the absolute URLs to the assets or run favicon checker by the deploy preview URL. @vince19972 

image

**Comment by vince19972** [comment]
Awesome! Same problem still exist on [Favicon checker](https://realfavicongenerator.net/favicon_checker?protocol=https&site=deploy-preview-70--cosmos-network.netlify.app%2F) though 🤔 I'll try to solve the problem. @lovincyrus

**Comment by lovincyrus** [comment]
> Same problem still exist on [Favicon checker](https://realfavicongenerator.net/favicon_checker?protocol=https&site=deploy-preview-70--cosmos-network.netlify.app%2F) though 🤔 I'll try to solve the problem.

Hmm, favicon checker is searching the wrong file path for `/img/icons/apple-touch-icon-152x152.png`. In fact, we don't even have the image file — `apple-touch-icon-152x152.png`

_RE_: apple-touch-icon.png and apple-touch-icon-precomposed.png
> It’s a good idea to include both apple-touch-icon.png and apple-touch-icon-precomposed.png for maximum compatibility: iOS 1 and BlackBerry OS6 don’t support precomposed icons. (Note that Android 2.1 only supports precomposed icons.)

`mask-icon` looks good to me too.

I think the only sensible next step is to duplicate an extra img file of apple-touch-icon and rename it with a suffix `-precomposed`

References: 
- https://mathiasbynens.be/notes/touch-icons
- http://markhagethorn.com/2015/11/23/apple_pinned_tab.html

**Comment by vince19972** [comment]
- Solved `apple-touch-icon-precomposed` by adding the file manually (not through Favicon Generator)

- `apple-touch-icon-152x152` is still ill-formed (from Favicon checker)
    - first failed attempt `<link rel="apple-touch-icon" sizes="152x152" href="<%= BASE_URL %>apple-touch-icon-152x152.png">`
    - second failed attempt `<link rel="apple-touch-icon" href="<%= BASE_URL %>apple-touch-icon-152x152.png">`
    - third failed attempt `<link rel="apple-touch-icon" href="<%= BASE_URL %>apple-touch-icon-100x100.png">`

- `Safari pinned tabs` is still ill-formed (from Favicon checker)
    - I've followed the [official guideline](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html) to flatten the `svg` file, using `16x16` viewBox, and pure black color, but still not working

**Comment by lovincyrus** [comment]
Let's remove the unused `apple-touch-icon-152x152.png` and merge this 🚢 @vince19972

File: public/site.webmanifest [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
-    "name": "Cosmos",
-    "short_name": "Cosmos",
+    "name": "",
```
Comment (lovincyrus) [inline public/site.webmanifest:2]: image

@vince19972 we should probably re-add the name too.

Comment (vince19972) [reply public/site.webmanifest]: ahhh sorry for missing it! Added and pushed.

**Pull request merged**